src/util/settingsMenu.js

```diff
@@ -181,12 +200,44 @@ const LoginPanel = ({
     }
   }
 
+  React.useEffect(() => {
+    if (!selectedProfile && profiles.length > 0) {
+      setSelectedProfile(profiles[0])
+    }
+  }, [selectedProfile, profiles])
+
   return <FeatureDependent requiredFeatures={[features.AUTHENTICATION]}>
     <ListItem sx={{ display: 'flex', justifyContent: 'center' }}>
       <Tooltip title={`logged in as ${token?.sub}`}>
         <Avatar/>
       </Tooltip>
     </ListItem>
+    {
+      profiles.length > 0 && <ListItem>
+        <FormControl variant='standard' sx={{width: '100%'}}>
+          <InputLabel>Profile</InputLabel>
+          <Select
+            value={selectedProfile}
+            label='Profile'
+            onChange={(e) => {
+              localStorage.setItem(PROFILE_KEY, e.target.value)
```

Comment:
I'd prefer to have a single point of local-storage write for profile selection (comment might be obsolete if deduplicated)